kimhongzhang323/Godamlah
# Frontend

This pull request includes several updates and improvements to the [Header.jsx] and Dashboard.jsx components, as well as the addition of new chart components. The changes aim to enhance the user interface and user experience by adding new features and fixing existing issues.

Changes Made
Header Component Updates:

Removed the message icon and added a notification icon with a popover containing fake notification content.
Cleaned up the code to remove any random or misplaced elements.
Added a "Settings" and "Sign out" option in the user menu with improved styling.
Dashboard Component Updates:

Added a "View All" button beside the "User Activity" heading to allow users to view all user activities.
Updated the layout to ensure the alerts and graph sections are displayed side by side with consistent styling.
Added custom scrollbars to the alerts and graph sections for a better user experience.
Chart Components:

Added SecurityDashboardBarChart component using @nivo/bar to display security metrics.
Added UserActivityBumpChart component using @nivo/bump to display user activity metrics.
Ensured the chart components are correctly imported and used in the Dashboard.jsx file.
Detailed Changes
Header.jsx:

Removed the message icon and related code.
Added a notification icon (HiOutlineBell) with a popover containing fake notification content.
Cleaned up the code to remove any random search button or misplaced elements.
Added "Settings" and "Sign out" options in the user menu with improved styling.
Dashboard.jsx:

Added a "View All" button beside the "User Activity" heading.
Updated the layout to ensure the alerts and graph sections are displayed side by side.
Added custom scrollbars to the alerts and graph sections.
Imported and used SecurityDashboardBarChart and UserActivityBumpChart components.
Chart Components:

Created SecurityDashboardBarChart.jsx using @nivo/bar to display security metrics.
Created UserActivityBumpChart.jsx using @nivo/bump to display user activity metrics.
Ensured the data for the charts is correctly imported and structured.
How to Test
Header Component:

Verify that the notification icon is displayed and the popover contains fake notification content.
Check that the "Settings" and "Sign out" options are present in the user menu and styled correctly.
Dashboard Component:

Verify that the "View All" button is displayed beside the "User Activity" heading.
Check that the alerts and graph sections are displayed side by side with consistent styling.
Ensure the custom scrollbars are working as expected.
Chart Components:

Verify that the SecurityDashboardBarChart displays the security metrics correctly.
Check that the UserActivityBumpChart displays the user activity metrics correctly.
Additional Notes
Ensure that all dependencies are installed by running npm install.
Verify that the application runs without errors and the new features work as expected.
